src/pages/index.js

```diff
@@ -1,15 +1,17 @@
 import * as React from "react"
 import Layout from "../components/Layout";
+import { graphql } from "gatsby"
+import Img from "gatsby-image"
 
 
 // markup
-const IndexPage = () => {
+const IndexPage = ({ data }) => {
     return (
         <main className="bg-gray-700 min-h-screen">
             <title>NovelRT</title>
             <Layout>
                 <div id="main-branding" className="flex flex-col items-center py-12 md:px-12 lg:px-36 xl:px-46 2xl:px-64 bg-gradient-to-b from-green-400 via-blue-500 to-indigo-600">
-                    <p className="text-7xl">img</p>
+                    <Img className="w-64" fluid={data.novelchan.childImageSharp.fluid} alt="Novel-Chan: NovelRT's mascot!" />
```

Comment:
Two things here:
1) Wasn't sure what to go with class-wise so I went with `w-64` to allow the picture to appear larger. This appears to respond well on mobile as well, so if you don't have any objections then I think this works for now.

2) Just note on how the fluid definition relates to the GraphQL query down below. 🙂

Reply:
It looks good on mobile and on desktop :)